receiver/README.md

```diff
@@ -180,3 +180,37 @@ receivers:
             static_configs:
               - targets: ['localhost:9777']
 ```
+
+## <a name="vmmetrics"></a>VM Metrics Receiver
+**Only metrics are supported.**
+
+<Add more information - I'm lonely.>
+
+## <a name="zipkin"></a>Zipkin Receiver
+**Only traces are supported.**
+
+This receiver receives spans from Zipkin (V1 and V2) HTTP uploads and translates them into the internal span types that are then sent to the collector/exporters.
+
+Its address can be configured in the YAML configuration file under section "receivers", subsection "zipkin" and field "address".  The syntax of the field "address" is `[address|host]:<port-number>`.
+
+For example:
+
+```yaml
+receivers:
+  zipkin:
+    address: "127.0.0.1:9411"
+```
+
+### Collector Differences
```

Comment:
This section doesn't apply anymore: it is all under a single configuration format and there is no cli option to enable the receivers.

Reply:
This is the same documentation as what is currently in master, just moved around to be alphabetical. The zipkin receiver documentation should be updated in a follow up PR.